.circleci/config.yml

```diff
@@ -666,6 +676,7 @@ workflows:
       - main
       - reviewers-and-zadmin
       - es-tests
+      - localization
```

Comment:
Is this meant to be temporary because we have no other way of testing the extraction process ? I am not sure it's worth running the extraction as part of the general test suite - or even the release process: it's an extra job that will slow things down.

Reply:
FWIW this job is not slowing anything down. Also defining it as a job gives us the flexibility to run it how and whenever we want. If the goal here is to automate the extraction process, it has to be a job. The only thing left to define is the trigger.

If you want. I can remove it from the list of jobs that run on PRs, but per the screenshots below, it will NEVER be the bottleneck and yes, for now, gives us visibility into what is happening.

Localization:
<img width="193" alt="image" src="https://github.com/mozilla/addons-server/assets/19595165/58b172fc-3490-4f2a-9d96-9b0ba1a69049">

Main:
<img width="185" alt="image" src="https://github.com/mozilla/addons-server/assets/19595165/2fda435d-80e1-4900-921a-2fbf89b53a4a">

LMK if you still think I should remove it and I will.

Reply:
IMO it would be good to have this run on PRs so a dev understands what changes to localization content are present in a PR. The current "git diff" is a bit too dull of a knife for that but this can easily be improved. I just don't see why that has to happen now.

Reply:
More so, in the event that a PR does something (we will find a way, we always do) that makes it impossible to extract locales... don't we want to know about it?

Reply:
Jobs run in parallel with a fixed number of jobs running at the same time, so any job can have an impact, we have to be careful. It seems fine with the current setup though.

There is a cost/CPU impact still, but I guess we can ignore that at least for now.

I am still not convinced about the value in PRs/commits though. At least until we can fix the formatting of our `.po` files once extracted to be more like Pontoon so that the `git diff` can be readable

Reply:
> More so, in the event that a PR does something (we will find a way, we always do) that makes it impossible to extract locales... don't we want to know about it?

That is a more convincing argument. I still don't like the overhead (maybe we should reorganize our CircleCI jobs) but let's try it.